dpnp/tests/test_mathematical.py

```diff
@@ -3320,6 +3320,30 @@ def test_matmul_with_offsets(self, sh1, sh2):
         expected = a[1] @ b[1]
         assert_dtype_allclose(result, expected)
 
+    def test_rmatmul_dpnp_array(self):
```

Comment:
That tests have to be part of `test_ndarray,py`, because they are testing method of ndarray.